### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs templating

##### ANSIBLE VERSION

```

```

### CODE REVIEW

**Comment by alikins** [comment]
Seeing some warnings on playbooks_templating.rst

```
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:9: ERROR: Content block expected for the "seealso" directive; none found.
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u':doc:`playbooks`'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    An introduction to playbooks'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u':doc:`playbooks_conditionals`'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    Conditional statements in playbooks'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u':doc:`playbooks_loops`'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    Looping in playbooks'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u':doc:`playbooks_roles`'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    Playbook organization by roles'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u':doc:`playbooks_best_practices`'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    Best practices in playbooks'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    Have a question?  Stop by the google group!'
/home/adrian/src/ansible/docs/docsite/rst/playbooks_templating.rst:12: WARNING: toctree contains reference to nonexisting document u'    #ansible IRC chat channel'
```

And playbooks_filters_ipaddr.rst isnt in an toc...
```
/home/adrian/src/ansible/docs/docsite/rst/playbooks_filters_ipaddr.rst:: WARNING: document isn't included in any toctree
```

**Comment by bcoca** [comment]
fixed the seealso and toctree for playbooks_template, filters_ipadr is not in toctree on purpose, it is linked from playbooks_filters, added `:orphan:` to remove that warning.

**Review by dharmabumstead — APPROVED** [review]
Approved with edits. Thanks.

**Comment by dharmabumstead** [comment]
Thanks @bcoca!

File: docs/docsite/rst/playbooks_templating.rst [hunk 1]
```diff
@@ -2,9 +2,9 @@ Templating (Jinja2)
 ===================
 
 As already referenced in the variables section, Ansible uses Jinja2 templating to enable dynamic expressions and access to variables.
-Not only that, it greatly expands the number of filters and tests available as well as adding it's own new plugin type, lookups.
+Jinja2 also greatly expands the number of filters and tests available, as well as adding a new plugin type: lookups.
```
Comment (bcoca) [inline docs/docsite/rst/playbooks_templating.rst:5]: Ansible .. not Jinja2

**Pull request merged**